saleor/graphql/product/types/products.py

```diff
@@ -797,17 +808,24 @@ class Collection(CountableDjangoObjectType):
     background_image = graphene.Field(
         Image, size=graphene.Int(description="Size of the image.")
     )
-    translation = TranslationField(CollectionTranslation, type_name="collection")
+    translation = TranslationField(
+        CollectionTranslation,
+        type_name="collection",
+        resolver=ChannelContextType.resolve_translation,
+    )
+    channel_listing = graphene.List(
```

Comment:
Not `channel_listings`? I see that we have the name `channel_listing`  which returns a list in more places on this branch, I guess we should change this name to plural everywhere @d-wysocki  @fowczarek? Also, the description here isn't very accurate. I would simply say "List of channels in which the collection is available".

Reply:
Good point, I will fix this in other PR. For now I will leave a comment, sounds good?

Reply:
👍